doc/src/drv_arch/drv_arch.md

```diff
@@ -82,6 +82,15 @@ user-space before calling the reconfiguration IOCTL.
 
 1. Unload the driver from the guest
 2. Unplug the VF from the guest
+
+.. note::
+
+```
+    NOTE: Unplugging the VF from the guest while an application on the guest is
+    still accessing its resources may lead to VM instabilities. We recommend
+    cloing all applications accessing the VF in the guest before unplugging the
```

Comment:
typo in `cloing`?
Also, what about rephrasing to:
```
It is recommended that all applications using a resource be closed prior to unplugging the VF.
```

Reply:
Cloing definitely sounds wrong. I'll fix it.